### PR TITLE
Support building on Xcode 8

### PR DESCRIPTION
**Pull request by stel**
/cc @TimOliver, @jpsim

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+    pod 'Realm'
     pod 'RealmConverter'
 
     target 'RealmBrowserTests' do
       # It looks like that inheritance via search paths is still broken with frameworks, see https://github.com/CocoaPods/CocoaPods/issues/4944
       # inherit! :search_paths
     end
 end
+
+# FIXME: Xcode 8 support
```
Comment (jpsim) [inline Podfile]: why is this segment needed? There's no Swift in this project.

**Comment by jpsim** [comment]
If we want to continue supporting Xcode 7.3.1 and Xcode 8 simultaneously, we should update the `.travis.yml` file accordingly. Something along the lines of this:

```yaml
language: objective-c
before_install: pod repo update
script: xcodebuild -workspace "RealmBrowser.xcworkspace" -scheme "Realm Browser" test CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO | xcpretty
matrix:
  include:
    - osx_image: xcode7.3
      env: JOB=Xcode7
    - osx_image: xcode8
      env: JOB=Xcode8
```

I haven't tested this^, so it might need modifications to work.

File: Podfile (hunk 1 above)
Comment (stel) [inline Podfile]: because of other dependencies like `CSwiftV` and `RealmConverter` that contain swift code

**Comment by stel** [comment]
Personally I think we don't need to support Xcode 7 anymore, so if everyone is ok with this I can modify both `.travis.yaml` and `Jenkinsfile` to use Xcode 8 also. /cc @radu-tutueanu

**Comment by jpsim** [comment]
Dropping Xcode 7 support is fine with me.

**Comment by stel** [comment]
All checks were passed, merging.

**Pull request merged**